File: Tarea07/ejercicio2.py
print("Ingresa 10 numeros")
f = [0,0,0,0,0,0,0,0,0,0]
for i in range(0, 10):
	f[i] = int(input())
print("Shift a la derecha")
for i in range(0, 9):
	print("%i, "%(f[i+1]), end='')
print(f[0])
print("Shift a la izquierda")
print("%i, "%(f[9]), end='')
for i in range(1, 10):
	print("%i, "%(f[i-1]), end='')
print('')

File: Tarea07/ejercicio2.c
#include <stdio.h>

int main(){
	int n, f[10];
	printf("Ingrsa 10 numeros\n");
	for (int n = 0; n < 10; ++n){
		scanf("%d", &f[n]);
	}
	printf("Shift a la derecha\n");
	for (n = 0; n < 9; ++n){
		printf("%i, ", f[n+1]);
	}
	printf("%i, ", f[0]);
	printf("\n");
	printf("Shift a la izquierda\n");
	printf("%i, ", f[9]);
	for (n = 1; n < 10; ++n){
		printf("%i, ", f[n-1]);
	}
	printf("\n");
	return 0;
}